packages/react-dom/src/shared/DOMProperty.js

```diff
@@ -206,6 +206,25 @@ export function getPropertyInfo(name: string): PropertyInfo | null {
   return properties.hasOwnProperty(name) ? properties[name] : null;
 }
 
+export function getCustomElementPropertyInfo(
+  name: string,
+  node: Element) {
+  if (name in (node: any)) {
+    const acceptsBooleans = (typeof (node: any)[name]) === 'boolean';
+    return {
```

Comment:
This seems a bit slow to do for every property. Likely will need some refactoring. However, is this check also equivalent on the built-ins?

Reply:
Are you referring to `name in (node: any)` or `(typeof (node: any)[name]) === 'boolean'`?